src/main/java/io/jenkins/plugins/appcenter/task/internal/DistributeResourceTask.java

```diff
@@ -99,8 +101,15 @@ private String parseReleaseNotesFromFile(@Nonnull String pathToReleaseNotes) {
 
     }
 
+    private static BuildInfo CreateBuildInfo(@Nonnull UploadRequest request) {
```

Comment:
I think we should change CreateBuildInfo to createBuildInfo as (in so far as I can tell) it is simply a private method of this class.... not a class itself.